site/docs/v1/tech/account-management/bootstrapping-login.adoc

```diff
@@ -0,0 +1,36 @@
+---
+layout: doc
+title: Bootstrapping Login For Self Service Account Management
+description: Allow users to access self-service account management from external applications
+---
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.45.0
+====
+
+== Accessing the Self Service Account Pages From External Applications
+
+In some scenarios you may want to be able to allow users to access the self-service account management pages when it is impractical to use a normal login flow. One such scenario is with native mobile applications where the user is logged in to FusionAuth and the application is in possession of the access token. The following procedure will allow you to access these pages without requiring the user to log in again.
+
+=== Opening the Self Account Pages With an Access Token
+
+In order to access the self-service account management pages in this manner you will need to meet the following criteria:
+
+* The user is logged in
+* The application is in possession of the access token JWT
+* The application can open a browser window or webview and include an Authorization header
```

Comment:
Is webview a generic term? Or should it be WebView?

Reply:
I think WebView is technically an Android thing but iOS people will know (it's like UIView in native iOS and otherwise dependant on what framework you chose to build both)